source/config-file/config-file-options.txt

```diff
@@ -59,6 +59,22 @@ You can configure the following settings in a configuration file:
      - Configure |compass| to not perform outgoing network operations other 
        than those to the database.
 
+Example 
+-------
+To view a sample |compass-short| configuration file, run the following command 
+in the folder containing your |compass| executable: 
+
+.. code-block:: sh
+
+     <path-to-Compass-executable> --show-example-config
+
+If you do not already have a configuration file, you can run the following 
```

Comment:
Consider shortening:

If you do not already have a configuration file, you can use the sample configuration file to create one:

Reply:
I'm leaning towards keeping this mostly as is (with some clarification tweaks) just to be a bit more explicit about what the command does.